keyboards/pisces/config.h

```diff
@@ -0,0 +1,69 @@
+/* Copyright 2021 knaruo
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x6B6E
+#define PRODUCT_ID   0x0000
+#define DEVICE_VER   0x0001
+#define MANUFACTURER knaruo
+#define PRODUCT      pisces
+
+#define USE_SERIAL
+
+/* serial.c configuration for split keyboard */
+#define SOFT_SERIAL_PIN D2
+
+/* Select hand configuration */
+#define SPLIT_HAND_MATRIX_GRID B0,B7
```

Comment:
You'll want to use matrix mask, if you're using the split grid things, so it doesn't keep the system awake. 

```suggestion
#define SPLIT_HAND_MATRIX_GRID B0,B7
#define MATRIX_MASKED
```

Reply:
@drashna 
Thank you for reviewing this. I applied MATRIX_MASKED.